app/src/test/java/com/yalta/repositories/FakePasswordRepo.kt

```diff
@@ -0,0 +1,16 @@
+package com.yalta.repositories
+
+class FakePasswordRepo : PasswordRepo {
+    private var canChangePassword = true
+
+    override suspend fun changePassword(newPassword: String): RepoResponse<PasswordChanged> {
+        val changed = canChangePassword
+        canChangePassword = !canChangePassword
```

Comment:
That's not how it should be. Have a look at mockito.

Or at least remove this weird behavior. 
This leads to tests of service being dependent on services dependency that is defined outside of tests.

Reply:
added